### PR TITLE
Inserted ifeval for note on architecture diagram

### PR DESCRIPTION
**Pull request by JoanneOFlynn2018**

### CODE REVIEW

**Comment by leifmadsen** [comment]
@pkilambi any suggestions on phrasing here?

File: doc-Service-Telemetry-Framework/modules/con_architecture.adoc [hunk 1]
```diff
@@ -37,6 +37,13 @@
 .Service Telemetry Framework architecture overview
 image::OpenStack_STF_Overview_37_1019_arch.png[]
 
+ifeval::["{build}" == "downstream"]
+
+[NOTE]
+The Service Telemetry Operator is fully supported. Prometheus  and ElasticSearch are third-party components.
```
Comment (pkilambi) [inline doc-Service-Telemetry-Framework/modules/con_architecture.adoc:43]: something along the lines of - "Service Telemetry Operator is fully supported in our Product. Though the services that are installed as part of this Operator such as Prometheus and ElasticSearch are community based Operators and their support is best effort. "

Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/con_architecture.adoc:43]:
```suggestion
The Service Telemetry Operator, Smart Gateway Operator and Smart Gateway are fully supported in the Service Telemetry Framework product, though the data storage components that are installed as part of this Operator (such as Prometheus and ElasticSearch) are community-based and their support is best-effort. 
```

Comment (leifmadsen) [reply doc-Service-Telemetry-Framework/modules/con_architecture.adoc:43]: @JoanneOFlynn2018 how about this? Feel free to tweak further. Thanks!

CC: @pkilambi

**Comment by leifmadsen** [comment]
[Update documentation to make it more clear what is fully-supported vs best-effort](https://trello.com/c/ccLVe9aK/981-update-documentation-to-make-it-more-clear-what-is-fully-supported-vs-best-effort)

File: doc-Service-Telemetry-Framework/modules/con_architecture.adoc [hunk 2]
```diff
@@ -37,6 +37,13 @@
 .Service Telemetry Framework architecture overview
 image::OpenStack_STF_Overview_37_1019_arch.png[]
 
+ifeval::["{build}" == "downstream"]
+
+[NOTE]
+The components of {Project} that comprise the data collection, collectd and  Ceilometer, and the transport, {MessageBus} and Smart Gateway, are fully supported. The data storage components, Prometheus and ElasticSearch, including the Operator artifacts, and visualization component Grafana are community-supported, and are not officially supported.
```
Comment (leifmadsen) [inline doc-Service-Telemetry-Framework/modules/con_architecture.adoc:43]:
```suggestion
The components of {Project} that comprise the data collection, collectd and Ceilometer, and the transport, {MessageBus} and Smart Gateway, are fully supported. The data storage components, Prometheus and ElasticSearch, including the Operator artifacts, and visualization component Grafana are community-based, and are not officially supported.
```

Comment (leifmadsen) [reply doc-Service-Telemetry-Framework/modules/con_architecture.adoc:43]: @JoanneOFlynn2018 you ok with these tweaks? If so, go ahead and merge :)

Comment (JoanneOFlynn2018) [reply doc-Service-Telemetry-Framework/modules/con_architecture.adoc:43]: Some useful comments in the peer review triggered a rewrite.

**Pull request merged**